sml_builder/templates/help-methods-request.html

```diff
@@ -0,0 +1,122 @@
+{%- extends "core.html" -%}
+{% from "components/section-navigation/_macro.njk" import onsSectionNavigation %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+{% import "related-content-macro.html" as macros %}
+
+{% set mainCols = 7 %}
+{% set asideCol =  {
+    "position": "after",
+    "colClasses": "ons-u-mt-l ons-u-mb-l ons-related-content ons-push-1@m"
+  }
+%}
+{%- set page_title = "Help centre" -%}
+{%- set current_path = url_for("help_centre") -%}
+{%- block main -%}
+  <h1>Help centre</h1>
+    <h2>{{category_label}}</h2>
+    <h3>{{sub_category_label}}</h3>
+    <h4>How do I submit new methods, or change existing methods, in the SML? (As an ONS Internal user)</h4>
+    <p>Method requests can be raised to:</p>
+        {{
+            onsList({
+                "variants": 'dashed',
+                "itemsList": [
+                    {
+                        "text": 'Have new methods developed for the Statistical Methods Library.'
+                    },
+                    {
+                        "text": 'Add existing code to the Statistical Methods Library.'
+                    },
+                    {
+                        "text": 'Change existing methods held within the Statistical Methods Library.'
+                    }
+                ]
+            })
+        }}
+    <p>Method requests can be submitted to the ONS Statistical Methods Library via the IBM Business Automation Workflow (BAW). The BAW is used across the ONS for a variety of purposes, so you may have access to this tool already.</p>
+    <p>If you do not have access to the IBM Business Automation Workflow (BAW): </p>
+        {{
+            onsList({
+                "element": 'ol',
+                "itemsList": [
+                    {
+                        "text": 'You can gain access to the BAW via ServiceNow in “I Want Something > 04 – Software, Applications & Platforms > BAW Support”.'
+                    },
+                    {
+                        "text": 'Select the option “Add A New BAW User”.'
+                    },
+                    {
+                        "text": 'Under “Process app name” enter “SML Method Request”.'
+                    },
+                    {
+                        "text": 'Fill out the other information as appropriate to you.'
+                    }
+                ]
+            })
+        }}
+    <p>If you already have access to the BAW, and want access to the SML forms: </p>
```

Comment:
This sounds wrong "the BAW" doesn't make sense to me - BAW as far as I know is an external tool used to create workflow applications. Obviously should match whatever we have been told, but we may need to query that.